packages/react-charts/src/components/ChartDonutUtilization/ChartDonutUtilization.tsx

```diff
@@ -596,8 +596,12 @@ export const ChartDonutUtilization: React.FunctionComponent<ChartDonutUtilizatio
       const donutThresholds = getDonutThresholds();
       const mergeThemeProps = (i: number) => {
         // Merge just the first color of dynamic (blue, green, etc.) with static (gray) for expected colorScale
-        newTheme.pie.colorScale[0] = donutThresholds[i].color;
-        newTheme.legend.colorScale[0] = donutThresholds[i].color;
+        if (newTheme.pie.colorScale instanceof Array) {
+          newTheme.pie.colorScale[0] = donutThresholds[i].color;
+        }
+        if (newTheme.legend.colorScale instanceof Array) {
+          newTheme.legend.colorScale[0] = donutThresholds[i].color;
+        }
```

Comment:
If `colorScale` is not an array, then the static color will never be set. We need to set a default, for example:

```
        if (newTheme.pie.colorScale instanceof Array) {
          newTheme.pie.colorScale[0] = donutThresholds[i].color;
        } else {
          newTheme.pie.colorScale = donutThresholds[i].color;
        }
```

Reply:
OK, since `string` is unchangeable so I will find a method to assign a value to `string`.

Reply:
I didn't realize `colorScale` was unchangeable?

Reply:
Sorry, my bad. It's changeable. Never mind.